### DISCUSSION TITLE
[Feature] iOS password autofill URL

### DISCUSSION THREAD

**Discussion by mmomjian** [immich-app/immich#9024] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Per discussion on Discord, the Android app can do password autofill using the URI `androidapp://app.alextran.immich`. It seems that there is not a similar URL for iOS, I have tested this by trying to auto fill a PW using BitWarden. Is there a URI that will work for the iOS app? 

It seems like a starting point would be: https://community.bitwarden.com/t/how-to-get-uri-for-ios-app/7544/11

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by nicomerl** [comment]
+1 
I am using Bitwarden on iOS and miss this Feature.

**Comment by burkemw3** [comment]
As mobile logouts continue to happen (#8146), missing password autofill is more painful.

**Comment by bo0tzz** [comment]
#3618
